superset/databases/commands/validate.py

```diff
@@ -90,7 +92,7 @@ def run(self) -> None:
 
         # try to connect
         sqlalchemy_uri = engine_spec.build_sqlalchemy_uri(
-            self._properties["parameters"],  # type: ignore
+            self._properties.get("parameters", None),  # type: ignore
```

Comment:
@betodealmeida I put some guards here against null values. Not sure if you want to catch them in an earlier validation or not. I also made a similar note about passing in port as a string that needs to be validated.

Reply:
We do validate the payload here: https://github.com/apache/superset/blob/master/superset/databases/api.py#L1006

But we don't validate the parameters' schema, since they are dynamic (though we could, it's just more complicated). And even if we did, adding a guard here is great!